_security/authentication-backends/jwt.md

```diff
@@ -0,0 +1,289 @@
+---
+layout: default
+title: JSON Web Token
+parent: Authentication backends
+nav_order: 47
+redirect_from:
+  - /security/configuration/configuration/
+---
+
+
+# JSON Web Token
+
+JSON Web Tokens (JWTs) are JSON-based access tokens that assert one or more claims. They are commonly used to implement single sign-on (SSO) solutions and fall in the category of token-based authentication systems:
+
+1. A user logs in to an authentication server by providing credentials (for example, a user name and password).
+1. The authentication server validates the credentials.
+1. The authentication server creates an access token and signs it.
+1. The authentication server returns the token to the user.
+1. The user stores the access token.
+1. The user sends the access token alongside every request to the service that it wants to use.
+1. The service verifies the token and grants or denies access.
+
+A JWT is self-contained in the sense that it carries within itself all of the information necessary to verify a user. The tokens are base64-encoded, signed JSON objects.
```

Comment:
It may be worth mentioning that it also contains information about the window its valid for - often they are not valid indefinitely but for the window of time of a session. Not sure if it's worth mentioning since this is an overview of JWTs for a newcomer. wdyt?

Reply:
@cwperks I think that's a good idea. I added some content about this as a last step in the ordered list.